src/mlpack/tests/q_learning_test.cpp

```diff
@@ -590,3 +591,38 @@ TEST_CASE("SACForMultipleActions", "[QLearningTest]")
   // If the agent is able to reach till this point of the test, it is assured
   // that the agent can handle multiple actions in continuous space.
 }
+
+//! Test PPO Discrete in CartPole task.
+TEST_CASE("CartPoleWithPPODiscrete", "[QLearningTest]")
+{
+  bool converged = false;
+  for (size_t trial = 0; trial < 4; ++trial)
```

Comment:
Does this pass on your system at least once?

Reply:
No. I can not understand as to it isn't converging. Will dig in more ! 
Do you have any ideas ?

Reply:
Let's discuss this today.